### PR TITLE
md5 deprecation

### PR DESCRIPTION
**Pull request by jeffbanks**
As part of #903, removing MD5 references for high priority security advisories.

### CODE REVIEW

**Comment by jeffbanks** [comment]
[Removal of endpoints as dead code](https://github.com/nosqlbench/nosqlbench/pull/909/commits/fbbe4f483427fa3a2d424cac46c3c1473706da30)
This is cleanup of dead code no longer utilized in nb5.

File: virtdata-userlibs/src/main/resources/docs-for-virtdata/bindings/funcref_conversion.md [hunk 1]
```diff
@@ -228,6 +228,7 @@ Convert the input value to a long.
 
 
 ## ToMD5ByteBuffer
+**[ DEPRECATED ]** no longer supported use DigestToByteBuffer with stronger digest algorithm.
```
Comment (jshook) [inline virtdata-userlibs/src/main/resources/docs-for-virtdata/bindings/funcref_conversion.md:231]: We should make the docs exporter do this for us in a separate issue.

**Review by jshook — APPROVED** [review]
Thanks for the thorough and defensive approach on this. Users will be happier.

File: virtdata-userlibs/src/main/resources/docs-for-virtdata/bindings/funcref_conversion.md (hunk 1 above)
Comment (jeffbanks) [reply virtdata-userlibs/src/main/resources/docs-for-virtdata/bindings/funcref_conversion.md:231]: I created a new docs issue to track that effort.

**Pull request merged**